docs/hiring-guide/community-spotlight/interview-lana-brindley.rst

```diff
@@ -0,0 +1,38 @@
+Interview with Lana Brindley
+=========================
+
+Lana is a technical writer at SUSE. Her work has primarily been with open source software, and she has been writing technical documentation and leading writing teams for 12 years.
+
+Key takeaways
+*************
+*Write whatever you can to build and show off your skills.
```

Comment:
These probably need a line before the heading, and a space after the asterisk to display properly.